README.md

```diff
@@ -1,2 +1,17 @@
 # 2017-Hestia
-A home automation app.
+A home automation system.
+
+## General Description 
+This repository contains the clientside code of the **Hestia** home automation system. The system consists of a server which has to be set up at home and an app.
+The server is used to manage all home automation devices. The app, which is available as an apk for the Android framework,
+connects to the server and provides an interface to control the devices. 
+
+The current code represents a Minimally Viable Product (MVP), which supports the Philips Hue system. We will extend this system in the near future to add support for more plugins. 
+
+## Setup
+To install the app, all that is needed is the apk. The apk can be obtained by pulling the current release (0.3) and then compiling it using Android Studio. 
```

Comment:
Agree, furthermore I think we should add the APK in the master branch possible, because the TA's / Mircea might not actually have android studio or whatever. It is easy to add the APK.